python/pyspark/ml/image.py

```diff
@@ -207,6 +207,9 @@ def readImages(self, path, recursive=False, numPartitions=-1,
         .. note:: If sample ratio is less than 1, sampling uses a PathFilter that is efficient but
             potentially non-deterministic.
 
+        .. note:: Deprecated in 2.4.0. Use `spark.read.format("image").load(path)` instead and
```

Comment:
Would it make sense to issue a warning when people call this code path (e.g. `warnings.warn`)? We do it in some of the other Python deprecated APIs but not all.

Reply:
added.